coredns/assets/configuration/spec.yaml

```diff
@@ -36,7 +36,7 @@ files:
         To enable CoreDNS metrics you must specify the openmetrics endpoint url
         and enable the plugin within coredns
         See: https://coredns.io/plugins/metrics/
-      required: false
+      enabled: true
```

Comment:
I set this to `enabled: true` to allow both `openmetrics_endpoint` and `prometheus_url`.